ae/core/autogen_wrapper.py

```diff
@@ -229,6 +229,12 @@ async def __create_user_delegate_agent(self) -> autogen.ConversableAgent:
         """
         def is_planner_termination_message(x: dict[str, str])->bool: # type: ignore
              should_terminate = False
+             print ("Checking Termination for Content:", x)
```

Comment:
Do we want to keep this print or was that more for debug

Reply:
Removed